src/components/selectable/selectable_list/selectable_list_item.tsx

```diff
@@ -51,6 +58,16 @@ export type EuiSelectableListItemProps = LiHTMLAttributes<HTMLLIElement> &
      * The default content when `true` is `↩ to select/deselect/include/exclude`
      */
     onFocusBadge?: boolean | EuiBadgeProps;
+    /**
+     * Padding for the list items.
+     */
+    paddingSize?: EuiSelectablePaddingSize;
+    /**
+     * Whether the `EuiSelectable` instance is searchable.
+     * When false, allows SPACE to toggle item selection.
+     * Enables the correct screen reader instructions for selection.
```

Comment:
Seriously appreciate the new inline comments, thanks for them! I have a suggestion for the docs here that would clear have cleared up some of my earlier confusion - lmk what you think. Not sure if it's too overkill but I really think it helps clarify a11y best practices
```suggestion
     * Whether the `EuiSelectable` instance is searchable.
     * When true, the Space key will not toggle selection, as it will type into the search box instead. Screen reader instructions will be added instructing users to use the Enter key to select items.
     * When false, the Space key will toggle item selection. No extra screen reader instructions will be added, as Space to toggle is a generally standard for most select/checked elements.
```